src/toolbar/ToolBar.js

```diff
@@ -119,7 +119,9 @@ class ToolBar extends React.Component{
             <i className="glyphicon glyphicon-trash"></i> Delete
           </button>:null;
     var searchTextInput = this.props.enableSearch?
-      <input type='text' placeholder={this.props.searchPlaceholder?this.props.searchPlaceholder:'Search'} onKeyUp={this.handleKeyUp.bind(this)}/>:null;
+      <div className="form-group form-group-sm">
+        <input className="form-control" type='text' placeholder={this.props.searchPlaceholder?this.props.searchPlaceholder:'Search'} onKeyUp={this.handleKeyUp.bind(this)}/>
```

Comment:
@imaireaux, id you want to use default search element in <code>react-bootstrap-table</code>, you can disable <code>search</code> and customize yours and call <code>handleSearch</code> on <code>BootstrapTable</code>, check the example on following:
```js

function search(text) {
    this.refs.table.handleSearch(text);
}

<BootstrapTable ref='table' data={products}>
    //...
</BootstrapTable>
```
See the <code>handleSearch</code> [docs](http://allenfang.github.io/react-bootstrap-table/docs.html)